### PR TITLE
Create cisco_ios_show_ap_summary.textfsm

### PR DESCRIPTION
**Pull request by Rishabh4290**
##### ISSUE TYPE

 - New Template Pull Request
 - Bugfix Pull Request
 - Additional Testing
 - Docs Pull Request

##### SUMMARY

```

```

### CODE REVIEW

**Comment by agentggg** [comment]
Hey there. I noticed the output does not clearly reflect the **show ap summary** command output on Cisco WLC. Any thoughts on how testing went or if it is a known issues, any thoughts on fixing this?

File: ntc_templates/templates/cisco_ios_show_ap_summary.textfsm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Value IP ([a-fA-F0-9:\.]+)
+Value State (\S+)
+
+Start
+  ^[a-zA-Z]+\s[a-z]+\s[a-zA-z]+:\s\d+$$
+  ^$$
+  ^AP\s+Name\s+Slots\s+AP\s+Model\s+Ethernet\s+MAC\s+Radio\sMAC\s+Location\s+Country\s+IP\s+Address\s+State\s*$$
+  ^-+$$
+  ^${AP_NAME}\s+${SLOT}\s+${AP_MODEL}\s+${MAC}\s+${RadioMac}\s+${LOCATION}\s+${COUNTRY}\s+${IP}\s+${State}\s+$$ -> Record
+  ^${AP_NAME}\s+${SLOT}\s+${AP_MODEL}\s+${MAC}\s+${RadioMac}\s+${LOCATION}\s+${COUNTRY}\s+${IP}\s+${State}$$ -> Record
```
Comment (jmcgill298) [inline ntc_templates/templates/cisco_ios_show_ap_summary.textfsm:17]:
```suggestion
  ^${AP_NAME}\s+${SLOT}\s+${AP_MODEL}\s+${MAC}\s+${RadioMac}\s+${LOCATION}\s+${COUNTRY}\s+${IP}\s+${State}\s*$$ -> Record
```

File: ntc_templates/templates/cisco_ios_show_ap_summary.textfsm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Value MAC ([a-fA-F0-9:\.]+)
+Value RadioMac ([a-fA-F0-9:\.]+)
+Value LOCATION (\S+\s\S+)
+Value COUNTRY (\S+)
+Value IP ([a-fA-F0-9:\.]+)
+Value State (\S+)
+
+Start
+  ^[a-zA-Z]+\s[a-z]+\s[a-zA-z]+:\s\d+$$
+  ^$$
```
Comment (jmcgill298) [inline ntc_templates/templates/cisco_ios_show_ap_summary.textfsm:13]:
```suggestion
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks @Rishabh4290 we will also need test files for this PR to be merged. Please see https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_parser.md#tests

**Comment by jmcgill298** [comment]
replaced by https://github.com/networktocode/ntc-templates/pull/1257

**Pull request closed without merging**